dataactbroker/handlers/accountHandler.py

```diff
@@ -100,7 +101,7 @@ def login(self,session):
                         if(self.interfaces.userDb.hasPermission(user, permission.name)):
                             permissionList.append(permission.permission_type_id)
                     self.interfaces.userDb.updateLastLogin(user)
-                    return JsonResponse.create(StatusCode.OK,{"message":"Login successful","user_id": int(user.user_id),"name":user.name,"title":user.title ,"agency":user.agency, "permissions" : permissionList})
+                    return JsonResponse.create(StatusCode.OK,{"message":"Login successful","user_id": int(user.user_id),"name":user.name,"title":user.title ,"cgac_code":user.cgac_code, "permissions" : permissionList})
```

Comment:
Would it be useful to whoever calls this function to also have the CGAC agency name included in the returned user info?